package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "@koopjs/provider-file-geojson",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
We don't increment our version on branches.  Instead, we merge to master, then do a commit on master with message `:package: <version>` - this helps us tag releases.